app/main/recipe_parser.py

```diff
@@ -171,6 +172,29 @@ def update_steps(self, file, steps):
         with open(file, 'w') as f:
             json.dump(updated_recipe, f, indent=4, sort_keys=True)
 
+def ZSeriesRecipeImport(recipe):
+    r = {}
+    r['name'] = recipe['Name']
+    name = recipe['Name']
+    current_app.logger.debug(f'DEBUG: saving recipe {name}')
+    r['clean'] = False
+    r['id'] = recipe['ID']

```

Comment:
Something that would likely show up for Z is a duplicate IDs - Pico's are 14 char/Zymatics are 32 char UUIDs, so the likely we auto generate a duplicate for them is small.  Maybe do a check if the ID already exists?